src/Error/Debug/HtmlFormatter.php

```diff
@@ -119,9 +119,10 @@ public function dump(NodeInterface $node): string
      * Convert a tree of NodeInterface objects into HTML
      *
      * @param \Cake\Error\Debug\NodeInterface $var The node tree to dump.
+     * @param int $indent The current indentation level.
      * @return string
      */
-    protected function export(NodeInterface $var): string
+    protected function export(NodeInterface $var, int $indent): string
```

Comment:
Wonder if we should default this to 0.

Reply:
I found the errors on undefined parameters were helpful as they made it really easy to track down where my mistakes are.

Reply:
I was just thinking for compatibility, but it's a protected function and no one is using it.